books/free-programming-books-pt_BR.md

```diff
@@ -231,6 +231,7 @@
 ### Inteligência Artificial
 
 * [Inteligência artificial: avanços e tendências](https://www.livrosabertos.sibi.usp.br/portaldelivrosUSP/catalog/view/650/579/2181) - Fabio G. Cozman, Guilherme Ary Plonski, Hugo Neri (PDF)
+* [Processamento de Linguagem Natural: Conceitos, Técnicas e Aplicações em Português](https://brasileiraspln.com/livro-pln/)
```

Comment:
Please add creator names where appropriate. https://brasileiraspln.com/livro-pln/FichaCatalogPLN.pdf

Reply:
Done, thanks for noticing!